src/web/pages/PowerBIReportsPage.tsx

```diff
@@ -1,78 +1,4 @@
 function PowerBIReportsPage() {
-    return (
```

Comment:
Why are you removing this? Shouldn't we keep this and remove the other one since the content isn't there?

Reply:
I only removed duplicate content left from the merge Arthur did. duplicate content and merge markers were not removed, but the page is there. Just checked my local copy.